<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_CustomTextWidget</class>
 <widget class="QWidget" name="Frame_CustomTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="TextContainerWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="Widget_TextCanvas" name="TextDisplay" native="true">
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollBar" name="TextverticalScrollBar">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="TexthorizontalScrollBar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normalon>:/G/Graphics/page_copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normalon>:/G/Graphics/paste_plain.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionClear_Screen">
   <property name="icon">
    <iconset>
     <normalon>:/G/Graphics/application.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>:/G/Graphics/zoom_in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normalon>:/G/Graphics/zoom_out.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionEndian_Swap">
   <property name="text">
    <string>Endian Swap</string>
   </property>
  </action>
  <action name="actionFind_CRC_Algorithm">
   <property name="text">
    <string>Find CRC Algorithm...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy_To_Send_Buffer">
   <property name="text">
    <string>Copy To Send Buffer...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_TextCanvas</class>
   <extends>QWidget</extends>
   <header>Widget_TextCanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
